cmake/Buildosqp-eigen.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) Fondazione Istituto Italiano di Tecnologia
```

Comment:
I guess this is a typo: filename should be `Buildosqp-matlab.cmake`, unless I missed something.

Reply:
Yes!